package.json

```diff
@@ -11,8 +11,9 @@
   "license": "MIT",
   "dependencies": {
     "iconv-lite": "^0.6.3",
+    "package.json": "^2.0.1",
```

Comment:
This shouldn't be in there, https://www.npmjs.com/package/package.json is intended for programmatically reading package json -- which we don't do.

Reply:
it was added automatically when I run npm install from code space but I can manually remove it.